e2e/paths/smoke_test.js

```diff
@@ -3,25 +3,7 @@ const config = require('../config.js');
 Feature('Smoke tests @smoke-tests').retry(2);
 
 Scenario('Sign in as local authority', (I, loginPage) => {
-  loginPage.signIn(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
-  I.see('Create new case');
-  I.signOut();
-});
-
-Scenario('Sign in as court admin', (I, loginPage) => {
```

Comment:
I removed these because we decided not to have these smoke tests users in production while in non prod environments login / logout is tested indirectly when access segregation is tested.